pkg/mynewt-core/contrib/callout.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2020 Inria
```

Comment:
some copyrights are "2020", is that up to date?

Reply:
Just moved the c file around, I would say the same copyright applies